crates/types/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "anvil_zksync_types"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[dependencies]
+zksync_types.workspace = true
+
+# TODO: Depending on `clap` here is not the worst thing for now but need to reconsider if this is strictly necessary
+clap.workspace = true
```

Comment:
It's quite surprising indeed.

Reply:
It's needed for clap-specific derivations. I could make it optional and hide it behind `clap` flag but at the same time I don't want to eventually arrive at the situation we have in zksync-era with the recently discussed feature flag hell.